composer.json

```diff
@@ -13,7 +13,7 @@
     "require": {
         "php": "^7.2.5",
         "laravel/scout": "^8.0",
-        "meilisearch/meilisearch-php": "^0.9|^0.10"
+        "meilisearch/meilisearch-php": "^0.9|^0.10|^0.11"
```

Comment:
@shokme. I'm not an expert in PHP, what does the `|` mean? Is it "or"?
Because the next version of meilisearch-laravel-scoot (v0.11) will only work with the `meilisearch-php` v0.11.

Reply:
Yes `|` is `OR.

Reply:
Ok so I don't accept this PR! Thanks @shokme